syntax = "proto3";

package op3;

// servicio op3
service Op3 {
	rpc Operation3 (Op3Request) returns (Op3Reply);
}

// mensaje solicitud
message Op3Request {
  float cateto1 = 1;
  float cateto2 = 2;
}

// mensaje respuesta
message Op3Reply {
  float hipotenusa = 1;
}
